e2e_test/batch/over_window/session/mod.slt.part

```diff
@@ -0,0 +1,61 @@
+# Because currently general streaming version of session window is not supported yet,
+# we only add e2e for batch mode.
+
+statement ok
+create table t (
+    tm timestamp,
+    foo int,
+    bar int
+);
+
+statement ok
+create view v1 as
+select
+    *,
+    first_value(tm) over (partition by bar order by tm session with gap '10 minutes') as window_start,
+    last_value(tm) over (partition by bar order by tm session with gap '10 minutes') as window_end
```

Comment:
Will these 2 window functions over the same window definition can be merged as a single `OverWindow` executor? Didn't see any test cases, but the backend implementation should support it. 

https://github.com/risingwavelabs/risingwave/blob/66df1d3754e3756cc33aa6d30a40f10c19c77bc8/src/stream/src/executor/over_window/general.rs#L58

Reply:
Yes, will be in the same OverWindow.

Reply:
Let's add a planner test for it?

Reply:
Actually we split OverWindow instances by `partition by` and `order by`, not by frame definitions (and we have planner test for this), so they should definitely in one OverWindow instances. But never mind, I just added this specific case as well.